src/csv_utils.py

```diff
@@ -0,0 +1,252 @@
+# Функции по работе с csv файлами:
+# Задание 10.08  (HW)
+
+# Чтение.
+
+
+def reading_csv(file_read: str) -> list:
+    """Открывает файл для чтения, формирует и возвращает список строк"""
+    import csv
+    with open(file_read, "r") as f:
+        finish_list_read = [row for row in csv.reader(f)]
+    return finish_list_read
+
+
+if __name__ == "__main__":
+    print(reading_csv("file1.csv"))
+
+
+# Запись (перезапись).
+
+
+def write_csv(input_list: list, file_name_write: str) -> None:
+    """Создает/открывает файл для записи и записывает в него входящий список по строкам"""
+    import csv
+    with open(file_name_write, "w") as f2:
+        csv_writer = csv.writer(f2, lineterminator="\n")
+        csv_writer.writerows(input_list)
+
+
+if __name__ == "__main__":
+    write_csv([["картина", "корзина", "собаченка"], [1, 2, 3, 4, 5]], "file11.csv")
+
```

Comment:
please create a one common if __name__ == '__main__' statement and put all your function calls there